### ISSUE TITLE
Repomd generates same SHA1 prefixes for all metadata files

### ISSUE THREAD

**Issue by g4s8** [artipie/rpm-adapter#29]
When using `NamingPolicy.Sha1Prefixed` with `Rpm`, `Repomd` generate same SHA1 prefixes for all metadata files. The prefix should be different and depend on files content.

**Comment by 0crat** [comment]
The job [#29](https://github.com/artipie/rpm-adapter/issues/29) assigned to @sammers21[/z](https://www.0crat.com/u/sammers21), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/46594759-3ff1-4bec-8b1f-5738d79dcef0); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); @sammers21[/z](https://www.0crat.com/u/sammers21) is not a member of this project yet, but they can request to join, as [§1](http://www.zerocracy.com/policy.html#1) explains; we should be aware that @sammers21[/z](https://www.0crat.com/u/sammers21) is on vacation; this ticket may be delayed; there will be no monetary reward for this job

**Comment by Sammers21** [comment]
In all the cases SHA1 prefix is equals to `da39a3ee5e6b4b0d3255bfef95601890afd80709`

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) I see that you unassigned this issue; the order is still assigned to @sammers21[/z](https://www.0crat.com/u/sammers21) though; to cancel the order use `refuse`, as in [§6](http://www.zerocracy.com/policy.html#6)

**Comment by 0crat** [comment]
The job [#29](https://github.com/artipie/rpm-adapter/issues/29) assigned to @g4s8[/z](https://www.0crat.com/u/g4s8), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/93dffc32-2343-4132-b3bb-167b1c7ba723); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be no monetary reward for this job

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@bf238f2eb839, thread: PQ-CT2E6TK9B
com.jcabi.xml.StrictXML[124] java.lang.IllegalArgumentException: 2 error(s) in XML document: -1:-1: cvc-pattern-valid: Value 'gh:Adapter back end integration' is not facet-valid with respect to pattern '[a-z]{2}:[A-Z0-9a-z.\-#/]+' for type 'milestoneID'.;-1:-1: cvc-complex-type.2.2: Element 'successor' must have no element [children], and the value must be valid.
```

[0.51.11](https://github.com/zerocracy/farm): CID: [d6b3513c-0acb-44be-a365-882b43cb9bb1](https://www.0crat.com/footprint/CT2E6TK9B/d6b3513c-0acb-44be-a365-882b43cb9bb1), Type: "Job milestoned"

**Comment by g4s8** [comment]
@rultor release, tag=`0.3.1`

**Comment by g4s8** [comment]
@rultor release, tag=`0.3.2`

**Comment by 0pdd** [comment]
@g4s8 the puzzle [#41](https://github.com/artipie/rpm-adapter/issues/41) is still not solved.

**Comment by 0crat** [comment]
Job was finished in 46 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0pdd** [comment]
@g4s8 2 puzzles [#60](https://github.com/artipie/rpm-adapter/issues/60), [#61](https://github.com/artipie/rpm-adapter/issues/61) are still not solved; solved: [#41](https://github.com/artipie/rpm-adapter/issues/41).

**Comment by 0pdd** [comment]
@g4s8 all 3 puzzles are solved here: [#41](https://github.com/artipie/rpm-adapter/issues/41), [#60](https://github.com/artipie/rpm-adapter/issues/60), [#61](https://github.com/artipie/rpm-adapter/issues/61).
